app/assets/stylesheets/application.scss

```diff
@@ -10,35 +10,24 @@ $govuk-use-legacy-font: false;
 // for guidance
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/components/back-link";
-@import "govuk_publishing_components/components/breadcrumbs";
-@import "govuk_publishing_components/components/button";
 @import "govuk_publishing_components/components/checkboxes";
 @import "govuk_publishing_components/components/details";
```

Comment:
Details can be removed from here (and `application.js`) as it isn't used by this application.